server/service/src/sync/translations/om_form_schema.rs

```diff
@@ -0,0 +1,96 @@
+use super::{
+    PullTranslateResult, PushTranslateResult, SyncTranslation, ToSyncRecordTranslationType,
+};
+use repository::{
+    ChangelogRow, ChangelogTableName, FormSchemaJson, FormSchemaRowDelete, FormSchemaRowRepository,
+    StorageConnection, SyncBufferRow,
+};
+// Needs to be added to all_translators()
+#[deny(dead_code)]
+pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
+    Box::new(OmFormSchemaTranslation)
+}
+pub(crate) struct OmFormSchemaTranslation;
+impl SyncTranslation for OmFormSchemaTranslation {
+    fn table_name(&self) -> &str {
+        "om_form_schema" // TODO should this be just form_schema? identifier is om_form_schema
+    }
+    fn pull_dependencies(&self) -> Vec<&str> {
+        vec![]
```

Comment:
I think we would need to add this table to om report dependencies

Reply:
lol just looked further down, looks like discussion already been had, i think ok to keep the dependency